app/api.py

```diff
@@ -372,18 +382,18 @@ def get_btc_dominance():
         return data["data"]["market_cap_percentage"]["btc"]
 
 
-class ByBtAPI:
+class CoinglassAPI:
 
-    base_url: str = "https://fapi.bybt.com:443"
+    base_url: str = "https://fapi.coinglass.com"
     headers = {
```

Comment:
do we really need header ?

Reply:
Not sure wa, just in case to prevent API blocking